patterns/search-big-heading.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Title: Search - Big heading and search form
+ * Slug: twentytwentyfive/search-big-heading
+ * Categories: call-to-action, text
+ * Description: heading with search form
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"align":"wide","style":{"spacing":{"blockGap":"var:preset|spacing|50","padding":{"top":"var:preset|spacing|80","bottom":"var:preset|spacing|80"}}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch"}} -->
+<div class="wp-block-group alignwide" style="padding-top:var(--wp--preset--spacing--80);padding-bottom:var(--wp--preset--spacing--80)"><!-- wp:heading {"level":1} -->
```

Comment:
The main heading or title of the template or page needs to be the only H1, so this block needs to use a different heading level.

On the blog template that shows the full content (the default home template), the page title is an H1 and and the post title is an H2.
So I recommend using H3 here.

Reply:
Hi @carolinan thank you for your review! I changed the heading from H1 to H3, I noticed on the design pattern that the font size is bigger, should I adjust the font size as well?

Reply:
Earlier today a new font size that matches the one in the design was added to theme.json.
You can pull down the latest changes from trunk to your fork and branch, and use the new size.

There is an example of how to use it here:
https://github.com/WordPress/twentytwentyfive/commit/8b711761eb7433e927b87dd361ce4f2d0ea8e038#diff-0e2183ba9203872d2f674be1dbc4134b15964cdd0cfddf972937f631dd0ce621R16
```
	<!-- wp:paragraph {"align":"center","style":{"typography":{"fontSize":"var(--wp--custom--font-size--huge)"}}} -->
	<p class="has-text-align-center" style="font-size:var(--wp--custom--font-size--huge);">
```
This is only for illustration purpose, you would add it to the heading.

Reply:
@carolinan The heading font size has been updated on this [commit](https://github.com/WordPress/twentytwentyfive/pull/166/commits/6470527b81c7eea85414bf5ef5593906ea26557d)